src/release.js

```diff
@@ -39,24 +39,29 @@ module.exports = async function ({ github, context, inputs }) {
   const { opticUrl, npmTag, version, id } = releaseMeta
 
   const run = runSpawn()
-  if (!pr.merged) {
-    const branchName = `release/${version}`
+  const branchName = `release/${version}`
+
+  try {
+    // We only delete the branch in case the release was successful
```

Comment:
I'm not sure I understand this comment. at this stage nothing has happened yet, so unless I'm missing something we are not actually deleting the branch only if the release is successful, we are doing it always

Reply:
Sorry, I rebased with a change that I forgot to push before.